internal/distro/rhel90/package_sets.go

```diff
@@ -115,12 +115,10 @@ func anacondaBootPackageSet(t *imageType) rpmmd.PackageSet {
 		ps = ps.Append(efiCommon)
 		ps = ps.Append(rpmmd.PackageSet{
 			Include: []string{
-				"grub2-efi-ia32-cdboot",
```

Comment:
For reference, and this is perfectly fine, the reason the ia32 grub2/shim gets pulled in by default there's been a number of very bad devices that are 64bit but with 32bit firmware and those are needed to boot it and then the kernel runs 64 bit. I doubt we'll need to ever support such a device but I'm adding this comment for reference.